source/Module/Rule.VsphereVssSecurity/Convert/Methods.ps1

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#region Method Functions
+
+<#
+    .SYNOPSIS
+        This function parses the fix text to find the boolean value of ForgedTransmits, then sets the value.
+
+    .PARAMETER RawString
+        An array of the raw string data taken from the STIG setting.
+#>
+function Get-VsphereForgedTransmits
+{
+    [CmdletBinding()]
+    [OutputType([object])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [psobject]
+        $RawString
+    )
+
+    if ($RawString -match 'Get-VirtualSwitch')
+    {
+        $VsphereForgedTransmits = ($RawString | Select-String -Pattern '(?<=ForgedTransmits\s)(.\w+)').Matches.Value
+    }
+
+    if ($null -ne $VsphereForgedTransmits)
+    {
+        Write-Verbose -Message $("[$($MyInvocation.MyCommand.Name)] Found ForgedTransmits value: {0}" -f $VsphereForgedTransmits)
+        return $VsphereForgedTransmits
+    }
+    else
+    {
+        return $null
+    }
+}
+<#
+    .SYNOPSIS
+        This function parses the fix text to find the boolean value of MacChanges, then sets the value.
+
+    .PARAMETER RawString
+        An array of the raw string data taken from the STIG setting.
+#>
+function Get-VsphereMacChanges
```

Comment:
guidelines for cmdlet/function nouns should be singular and not plural.

Reply:
This is based on the property from Vmware.VsphereDSC    

 .DESCRIPTION
    The flag to indicate whether or not the Media Access Control (MAC) address
    can be changed.
    #>
    [DscProperty()]
    [nullable[bool]] $MacChanges


https://github.com/vmware/dscr-for-vmware/blob/master/Source/VMware.vSphereDSC/VMware.vSphereDSC.psm1

Reply:
Yeah cmdlet nouns should be singular and so should parameter names.  It doesn't matter is someone else didn't follow the guidance we should do our best to follow it.